AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -365,6 +388,30 @@ - (void)setImage:(UIImage *)image forState:(ASControlState)state
   [self updateImage];
 }
 
+- (UIImage *)backgroundImageForState:(ASControlState)state
+{
+    ASDN::MutexLocker l(_propertyLock);
+    switch (state) {
+      case ASControlStateNormal:
+        return _normalBackgroundImage;
+            
+      case ASControlStateHighlighted:
+        return _highlightedBackgroundImage;
+            
+      case ASControlStateSelected:
+        return _selectedBackgroundImage;
+            
+      case ASControlStateSelected | ASControlStateHighlighted:
+        return _selectedHighlightedBackgroundImage;
+      
+      case ASControlStateDisabled:
+        return _disabledBackgroundImage;
+            
+      default:
+        return _normalBackgroundImage;
+    }
+}
+
```

Comment:
@aaronschubert0 whoops, the spacing was right but must've gotten blown away after I moved it up to match the order of other getter/setter combos. Fixed.